pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -37,30 +42,43 @@ import (
 	"knative.dev/pkg/network"
 	"knative.dev/pkg/reconciler"
 	"knative.dev/pkg/resolver"
+	v1 "knative.dev/serving/pkg/apis/serving/v1"
 	"knative.dev/serving/pkg/apis/serving/v1alpha1"
 	domainmappingreconciler "knative.dev/serving/pkg/client/injection/reconciler/serving/v1alpha1/domainmapping"
 	"knative.dev/serving/pkg/reconciler/domainmapping/config"
 	"knative.dev/serving/pkg/reconciler/domainmapping/resources"
+	routeresources "knative.dev/serving/pkg/reconciler/route/resources"
```

Comment:
Can we move this somewhere common?

Reply:
I'll follow up with that as well.

Reply:
Got an issue with a DomainMapping:
```
party       knative.party                                Unknown   NewObservedGenFailure
```

Looking at the logs of the kcert:

```
  Warning  InternalError  3m23s (x17 over 8m51s)  certificate-controller  Service "knative.party" is invalid: metadata.name: Invalid value: "knative.party": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')```

Reply:
I believe you need to update net-http01, let's discuss in slack